src/sif/pedestriancost.cc

```diff
@@ -59,7 +59,8 @@ constexpr float kDefaultSideWalkFactor = 1.0f;     // Neutral value for sidewalk
 constexpr float kDefaultAlleyFactor = 2.0f;        // Avoid alleys
 constexpr float kDefaultDrivewayFactor = 5.0f;     // Avoid driveways
 constexpr float kDefaultUseFerry = 1.0f;
-constexpr float kDefaultUseTracks = 0.5f; // Factor between 0 and 1
+constexpr float kDefaultUseTracks = 0.5f;        // Factor between 0 and 1
+constexpr float kDefaultUseLivingStreets = 0.6f; // Factor between 0 and 1
```

Comment:
should this be 1.f? isnt the point of most of these living streets that pedestrians get right of way at or above other modes of travel?

Reply:
I agree with your idea, but in my opinion pedestrians are more sensitive to route distance/duration increase which can happen if we favor living streets more. That's why I didn't want to mess with this costing a lot

Reply:
Interesting, so a customer who would like more scenic and clean routes may want to crank this up at the potential expense of time and distance. Makes sense.